contrib/local-environment/oauth2-proxy-traefik.cfg

```diff
@@ -0,0 +1,23 @@
+http_address="0.0.0.0:4180"
+cookie_secret="OQINaROshtE9TcZkNAm-5Zs2Pv3xaWytBmc5W7sPX7w="
+email_domains=["example.com"]
+cookie_secure="false"
+# upstreams="http://httpbin"
```

Comment:
Should probably remove this before merging

Reply:
:+1: